udf/anomaly-detection/src/udf/preprocess.py

```diff
@@ -22,27 +22,17 @@
 class Preprocess:
     @classmethod
     def get_df(cls, data_payload: dict, features: List[str]) -> (pd.DataFrame, List[int]):
-        timestamps = range(
+        ts = list(range(
```

Comment:
Do we have unit tests to verify the logic?

Reply:
yes we do have unit tests covering this logic